### DISCUSSION TITLE
Welcome to Appwrite Discussions!

### DISCUSSION THREAD

**Discussion by eldadfux** [appwrite/appwrite#759] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by punitdiwan** [comment]
Hi @eldadfux, I am Punit Diwan from India.  I am excited to see GitHub started forum. I look forward for this as it will be a great place to get support all users. For all Appwriters, all code, Issues,  PRs, discussions are under one roof.

**Comment by Glide7** [comment]
Hello everyone! 
Lior here from the Appwrite team - looking forward to hearing your suggestions and ideas! 💪

**Comment by codeByunique** [comment]
Hey everyone! I'm Atabul from India 🇮🇳  
I'm a Java & Spring Boot developer, currently exploring open-source.  
Happy to be here and looking forward to learning and contributing to this amazing community! 🚀
